source/storage/apis-1.5.rst

```diff
@@ -432,6 +509,47 @@ Request Headers
     response will be returned.
 
 
+**X-Weave-Records**
+
+    This header may be sent with multi-record uploads, to indicate the
+    total number of records included in the request.  If the server
+    would not accept an upload containing that many records, then a
+    **400 Bad Request** response will be returned with response code **17**.
+
+
+**X-Weave-Bytes**
+
+    This header may be sent with multi-record uploads, to indicate the
+    combined size of payloads in the upload, in bytes.  If the server
+    would not accept an upload containing that many bytes, then a
+    **400 Bad Request** response will be returned with response code **17**.
+
+
+**X-Weave-Batch-Records**
+
+    This header may be included with a POST request using the **batch** query
+    parameter, to indicate the total number of records in the batch.
```

Comment:
In my head a "batch" is a series of consecutive POST requests that will committed as a single unit, so "batch is approximately the same as "total".  Would it help if I said "in the entire batch"?  Alternately, s/Batch/Total/ here and below to avoid confusion?